wallet/wallet.go

```diff
@@ -498,6 +498,11 @@ out:
 				continue
 			}
 			timeout = req.lockAfter
+			if timeout == nil {
+				log.Info("Wallet unlocked without timeout")
+			} else {
+				log.Info("Wallet temporarily unlocked")
```

Comment:
There's no way to know how long the timeout is.  It's a channel, not a duration.